Home/index.html

```diff
@@ -47,18 +55,192 @@
         </div>
         <div class="hero_text">
             <h1>
-                Hey there. <br>
+                Hey there. <br />
                 I'm Luca
             </h1>
-            <h4>
-                I'm a Software Developer
-            </h4>
+            <h4>I'm a Software Developer</h4>
             <p>
                 I can help you build a product , feature or website Look through some
                 of my work and experience! If you like what you see and have a project
                 you need coded, don't hestiate to contact me.
             </p>
         </div>
     </section>
+
+    <section class="recent_works_container">
+        <div>
+            <h1>My Recent Works</h1>
+            <div class="line"></div>
+        </div>
+        <figure class="mps">
+            <img src="../img/bg/Placeholder.png" alt="Placeholder" />
+            <figcaption>
+                <h1>Muli-Post Stories</h1>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard dummy text ever
+                    since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </figcaption>
+        </figure>
+        <div class="works_grid_container">
+            <div class="card_hovered">
+                <button type="button">See Project</button>
+            </div>
+
```

Comment:
Please fix the bug in this card:
it overlaps the previous section

<img width="426" alt="image" src="https://user-images.githubusercontent.com/7243132/170836837-307e046d-969f-49a4-b01a-db03c867984a.png">